src/components/PageLayout.js

```diff
@@ -72,15 +78,13 @@ function DashBlock(props) {
     const rest = Object.assign({}, { ...other }, { className: newClassName });
 
     return (
-        <Segment basic {...rest} >
-            {(header) ?
+        <Segment basic {...rest}>
+            {header ? (
                 <div className="dashblock__head">
                     <Header as="h2" content={header} />
-                </div> : null
-            }
-            <div className="dashblock__content">
-                {children}
-            </div>
+                </div>
+            ) : null}
```

Comment:
@reyrodrigez 
Do not use the ternary operator to check variable if no false branch.
(https://github.com/Augmint/augmint-web/commit/7bfd35875bdfe5f7189b9e101f59cd39b35b8f62)
`{header && <Header as="h2" content={header} />}`